frontend/test/pytest/test_autograph.py

```diff
@@ -1756,6 +1758,50 @@ def g():
         assert g() == 36.4
 
 
+class TestAutographInclude:
+    """Test include modules to autograph conversion"""
+
+    def test_dummy_func(self):
+        """Test dummy function branches."""
+
+        assert dummy_func(6) == 36
+        assert dummy_func(4) == 64
+
+    def test_autograph_included_module(self):
+        """Test autograph included module."""
+
+        @qjit(autograph=True)
+        def excluded_by_default(x: float, n: int):
+            for _ in range(n):
+                x = x + dummy_func(6)
+            return x
+
+        @qjit(autograph=True, autograph_include=["catalyst.utils.dummy"])
+        def included(x: float, n: int):
+            for _ in range(n):
+                x = x + dummy_func(6)
+            return x
+
+        result_excluded_by_default = excluded_by_default(0.4, 6)
+        assert result_excluded_by_default == 216.4 and result_excluded_by_default == included(
+            0.4, 6
+        )
```

Comment:
@rauletorresc while this is testing that the outputs are the same, it isn't testing that one has been properly converted with autograph, and the other hasn't

Reply:
Such tests are perform as a lit tests: 

- https://github.com/PennyLaneAI/catalyst/blob/62331822a3026a9c4066315867fcce3f0a666c82/frontend/test/lit/test_autograph.py#L676
- https://github.com/PennyLaneAI/catalyst/blob/62331822a3026a9c4066315867fcce3f0a666c82/frontend/test/lit/test_autograph.py#L693

Reply:
thanks!